Hardware/Hardware/Printer/AirPrintReceipt/ReceiptRenderer.swift

```diff
@@ -20,34 +33,150 @@ final class ReceiptRenderer: UIPrintPageRenderer {
     }
 
     override func drawHeaderForPage(at pageIndex: Int, in headerRect: CGRect) {
-        let pageNumberString = NSString(string: "Order receipt. Page \(pageIndex + 1)")
-        pageNumberString.draw(in: headerRect, withAttributes: attributes)
+        guard let storeName = parameters.storeName else {
```

Comment:
I found storeName slightly confusing. I had to think about it as to whether this was a Storage reference or the merchant's store. Unfortunately we use both in our code, although we tend to use siteName for code (and storeName in tests)

Please consider using siteName instead as it seems less ambiguous?